### PR TITLE
Add support for BPF_MAP_TYPE_BLOOM_FILTER

### PR DESCRIPTION
**Pull request by nak3**
This patch adds support for `BPF_MAP_TYPE_BLOOM_FILTER`.

Close https://github.com/aya-rs/aya/issues/204

### CODE REVIEW

**Comment by nak3** [comment]
https://github.com/nak3/aya-showcase/tree/main/bloom-filter demonstrates this fix.

**Review by dave-tucker — APPROVED** [review]
Excellent work! LGTM :palm_tree:

File: aya/src/maps/bloom_filter.rs [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+        let _ = map.fd_or_err()?;
+
+        Ok(BloomFilter {
+            inner: map,
+            _v: PhantomData,
+        })
+    }
+
+    /// Query the existence of the elelment.
```
Comment (alessandrod) [inline aya/src/maps/bloom_filter.rs:67]: Typo: element.

Also I think this should take `value: &V`?

**Review by alessandrod — COMMENTED** [review]
Looks great, see comment!

**Pull request merged**